public/client.js

```diff
@@ -1,9 +1,30 @@
-// $(function () {
-// 	$.get('/trucks', function (trucks) {
-// 		var truckList = [];
-// 		trucks.forEach(function (truck) {
-// 			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
-// 		});
-// 		$('.truck-list').append(truckList);
-// 	});
-// });
\ No newline at end of file
+ $(function () {
+ 	$.get('/trucks', function (trucks) {
+ 		var truckList = [];
+ 		trucks.forEach(function (truck) {
+ 			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
+ 		});
+ 		$('.truck-list').append(truckList);
+ 	});
+     
+     
+//     i tried the extra credit as follow, but it's not working
+     
+//    $('#myDiv').click(function() {
```

Comment:
Because there is a link in this div, you will need to declare a parameter for the handler on which you can call preventDefault():

`$('#myDiv').click(function (e) { e.preventDefault();...}`

If you don't preventDefault, it will simply reload the page before the rest of the code executes.